hardware/esp8266com/esp8266/libraries/ESP8266WiFi/src/WiFiClient.cpp

```diff
@@ -300,3 +300,27 @@ void WiFiClient::stopAll()
         }
     }
 }
+
+
+void WiFiClient::stopAllexcepted(WiFiClient * exC) {
+    for (WiFiClient* it = _s_first; it; it = it->_next) {
+        ClientContext* c = it->_client;
+
+        if(c && exC->_client) {
```

Comment:
Can it be simply `if (exC->_client == it->_client)`? As far as i can tell, pcb is unique for all WiFiClient instances corresponding to the same connection.